lib/rust/ensogl/pack/src/assets.rs

```diff
@@ -0,0 +1,362 @@
+use crate::Paths;
+use enso_prelude::anyhow;
+use ide_ci::prelude::*;
+use ide_ci::programs::shaderc::Glslc;
+use ide_ci::programs::shaderc::SpirvOpt;
+use ide_ci::programs::spirv_cross::SpirvCross;
+use std::hash::Hasher;
+
+
+
+// =============
+// === Build ===
+// =============
+
+/// Build the dynamic assets.
```

Comment:
What does "build" mean? Could you please make the docs a little bit more wordy here please? :)